lib/mapper.js

```diff
@@ -113,7 +113,7 @@ function formatConversionTag(msg) {
   var ret = {
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
```

Comment:
https://developers.google.com/app-conversion-tracking/ios/remarketing-server

"You must indicate whether the device has "limit ad tracking" enabled. If so, set lat=1. If limit ad tracking is not enabled, set lat=0."